src/nsfwServer.js

```diff
@@ -10,65 +10,66 @@ const express = require('express');
 const cors = require('cors');
 const pino = require('pino-http')();
```

Comment:
Express and IPFS are using [debug](https://www.npmjs.com/package/debug) for logging. If we use that here too, it might create a more consistent debug output, as we're currently getting both intermixed:

![image](https://user-images.githubusercontent.com/22742/162744765-1f968962-aa20-4e64-9d9c-dfd07af02ad6.png)

Reply:
I don't know about ipfs, but express has no standard logging library. To my knowledge, pino and winston are the most popular. I tried what you said, using Debug, but there was no output, so I gave up. 
If we start running into real issues with it, we can define a spec of what we need from logging and build it. Until then I suggest, let's not bother. You can also try to see if you can fix it the way you prefer.

Reply:
Well I looked better at the picture now; on my dev terminal, I don't get any output like that from express, but I can just remove pino if preferred.